pyproject.toml

```diff
@@ -70,7 +70,6 @@ dependencies = [
     "httpx-ws>=0.6.0",
     "aiosqlite>=0.20.0",
     "InquirerPy>=0.3.4",
-    "uv",
```

Comment:
It is also required for locking the packages, should handle the missing uv binary there.